### ISSUE TITLE
say that sha1 is used for hash

### ISSUE THREAD

**Issue by schenlap** [pixlsus/raw#37]
currently unter raw there is only 

>
 pic_1-1.RWL
9d83454a250dc18ca59b9b842d834a6b77840b14(11.97MB)
> 
without md5, sha1, ...

**Comment by LebedevRI** [comment]
#5, i guess @patdavid removed it later on.
sha1 should be an alt text for the hashsum itself
